Formula/afl-fuzz.rb

```diff
@@ -15,7 +15,7 @@ def install
     # behaving correctly!" in a nested login shell.
     # Reported to lcamtuf@coredump.cx 6th Apr 2016.
```

Comment:
Could you try to submit a patch to them? It doesn't look like they are working on fixing this.